test/spec/gridfs/delete.json

```diff
@@ -49,10 +49,7 @@
           "uploadDate": {
             "$date": "1970-01-01T00:00:00.000Z"
           },
```

Comment:
These needed to be synced. We hadn't done https://jira.mongodb.org/browse/NODE-6581 yet.